test/built-ins/Array/of/creates-a-new-array-from-arguments.js

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2015 the V8 project authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+/*---
+es6id: 22.1.2.3
+description: >
+  Array.of method creates a new Array with a variable number of arguments.
+info: >
+  22.1.2.3 Array.of ( ...items )
+
+  ...
+  7. Let k be 0.
+  8. Repeat, while k < len
+    a. Let kValue be items[k].
+    b. Let Pk be ToString(k).
+    c. Let defineStatus be CreateDataPropertyOrThrow(A,Pk, kValue).
+    d. ReturnIfAbrupt(defineStatus).
+    e. Increase k by 1.
+  9. Let setStatus be Set(A, "length", len, true).
+  10. ReturnIfAbrupt(setStatus).
+  11. Return A.
+includes: [compareArray.js]
+---*/
+
+var a1 = Array.of('Mike', 'Rick', 'Leo');
+assert(
+  compareArray(a1, ['Mike', 'Rick', 'Leo']),
```

Comment:
Between the two different approaches this test uses to compare arrays, I prefer. the second. It doesn't rely on an external function or iteration semantics. It's more characters to type, but we don't expect this test to change any time soon :) Either approach is technically correct, so I think the most important thing is to consistently use one or the other.